### ISSUE TITLE
Investigate unit tests failing when S3 storage enabled

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#2280]
`ConnectionError: HTTPConnectionPool(host='testserver', port=80): Max retries exceeded with url: /private-media/someuser/exports/%D1%82%D1%94%D1%95%D1%82%20%CE%B1%D1%95%D1%95%D1%94%D1%82%20-%20all%20versions%20-%20labels%20-%202019-05-16-21-22-12.csv (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa8904e41d0>: Failed to establish a new connection: [Errno -2] Name or service not known',))`

Appeared with https://github.com/kobotoolbox/kpi/commit/55d9ea247?
Discussion: https://www.flowdock.com/app/kobotoolbox/kobo/threads/nW0QrhJAn1B0neeRISbq7d6IcTm

**Comment by jnm** [comment]
Workaround was https://github.com/kobotoolbox/kpi/pull/2186/files#diff-53d03734afc641f6aee8cc005fd1221cR11
```python
if 'KPI_AWS_STORAGE_BUCKET_NAME' in os.environ:
    PRIVATE_STORAGE_S3_REVERSE_PROXY = False
```

**Comment by noliveleger** [comment]
Fixed by #2603
